eth-bytecode-db/eth-bytecode-db/src/search/types.rs

```diff
@@ -1,8 +1,48 @@
 use bytes::Bytes;
-use entity::sea_orm_active_enums::BytecodeType;
+use entity::sea_orm_active_enums;
+use std::fmt::{Display, Formatter};
 
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct BytecodeRemote {
     pub bytecode_type: BytecodeType,
     pub data: Bytes,
 }
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum BytecodeType {
+    CreationCode,
+    RuntimeCode,
+    CreationCodeWithoutConstructor,
+}
+
+impl From<sea_orm_active_enums::BytecodeType> for BytecodeType {
+    fn from(value: sea_orm_active_enums::BytecodeType) -> Self {
+        match value {
+            sea_orm_active_enums::BytecodeType::CreationInput => BytecodeType::CreationCode,
+            sea_orm_active_enums::BytecodeType::DeployedBytecode => BytecodeType::RuntimeCode,
+        }
+    }
+}
+
+impl From<BytecodeType> for sea_orm_active_enums::BytecodeType {
+    fn from(value: BytecodeType) -> Self {
+        match value {
+            BytecodeType::CreationCode | BytecodeType::CreationCodeWithoutConstructor => {
+                sea_orm_active_enums::BytecodeType::CreationInput
+            }
+            BytecodeType::RuntimeCode => sea_orm_active_enums::BytecodeType::DeployedBytecode,
+        }
+    }
+}
+
+impl Display for BytecodeType {
```

Comment:
nit: in scoutcloud I decided to use https://docs.rs/serde_plain/latest/serde_plain/macro.derive_display_from_serialize.html
maybe you also want to use it :)

Reply:
Thanks! I will make use of it if such one more such enum arises or this one is changed frequently. But for now, I prefer leave it as it is, so not to add one more dependency just to save a couple of lines already written